views/js/qtiCreator/widgets/static/object/states/Active.js

```diff
@@ -13,188 +13,120 @@
-                media.removeAttr(propertyName);
-            }
-            $media.trigger('contentChange.qti-widget');
-
-        }, 100);
-
-    };
+    var refreshRendering = _.throttle(function refreshRendering(widget){
+        var obj = widget.element;
+        var $container = widget.$original;
+        var previewOptions = {
+            url : obj.renderer.resolveUrl(obj.attr('data')),
+            mime : obj.attr('type')
+        };
+        if(obj.attr('height')){
+            previewOptions.height = obj.attr('height');
+        }
+        if(obj.attr('width')){
+            previewOptions.width = obj.attr('width');
+        }
+        if(previewOptions.url && previewOptions.mime){
+            $container.previewer(previewOptions);
+        }
+    }, _config.renderingThrottle);
 
     ObjectStateActive.prototype.initForm = function(){
 
         var _widget = this.widget,
-            $media = _widget.$original,
             $form = _widget.$form,
-            media = _widget.element,
-            baseUrl = _widget.options.baseUrl,
-            responsive = true;
+            qtiObject = _widget.element,
+            baseUrl = _widget.options.baseUrl;
 
         $form.html(formTpl({
             baseUrl : baseUrl || '',
-            data : media.attr('data'),
-            height : media.attr('height'),
-            width : media.attr('width'),
-            responsive : responsive
+            src : qtiObject.attr('data'),
+            alt : qtiObject.attr('alt'),
+            height : qtiObject.attr('height'),
+            width : qtiObject.attr('width')
         }));
 
-        //init slider and set align value before ...
-        _initAdvanced(_widget);
-        _initSlider(_widget);
-        _initAlign(_widget);
+        //init resource manager
         _initUpload(_widget);
 
-        //... init standard ui widget
+        //init standard ui widget
         formElement.initWidget($form);
 
         //init data change callbacks
-        formElement.setChangeCallbacks($form, media, {
-            data : function(media, value){
-
-                media.attr('data', value);
-
-                if(!value.match(/^http/i)){
-                    value = baseUrl + '/' + value;
-                }
-                $media.attr('data', value);
-
+        formElement.setChangeCallbacks($form, qtiObject, {
+            src : function(object, value){
+                qtiObject.attr('data', value);
                 inlineHelper.togglePlaceholder(_widget);
-                _initSlider(_widget);
-                _initAdvanced(_widget);
+                refreshRendering(_widget);
             },
-            alt : formElement.getAttributeChangeCallback(),
-            longdesc : formElement.getAttributeChangeCallback(),
-            align : function(media, value){
-                inlineHelper.positionFloat(_widget, value);
+            width : function(object, value){
+                var val = parseInt(value, 10);
+                if(_.isNaN(value)){
```

Comment:
then you need 
var val = parseInt(value, 10);
+                if(_.isNaN(**val**)){

and not 
var val = parseInt(value, 10);
+                if(_.isNaN(**value**)){